docs/API/NuGet-Client-SDK.md

```diff
@@ -0,0 +1,32 @@
+---
+# required metadata 
+
+title: NuGet v3 Client and NuGetGallery APIs | Microsoft Docs
+author: kraigb
+ms.author: kraigb
+manager: ghogen
+ms.date: 1/9/2017
+ms.topic: article
+ms.prod: nuget
+ms.technology: null
+ms.assetid: d1a393b7-51b1-4840-b1a8-fdd76455077d
+
+# optional metadata
+
+description: The NuGet and NuGetGallery APIs are evolving and not yet documented, but examples are available on Dave Glick's blog.
+keywords: NuGet API, NuGetGallery API, NuGet v3 libraries
+ms.reviewer:
+- karann-msft
+- unniravindranathan
+
+---
+
+# NuGet Client SDK
+
+The NuGet v3 Client and NuGetGallery APIs are constantly evolving and we are working on having a stable surface area that we can document soon.
+
+In the meantime, you can find examples and documentation for some of the API in the following blog series by Dave Glick:
+
+- [Exploring the NuGet v3 Libraries, Part 1: Introduction and concepts](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-1)
+- [Exploring the NuGet v3 Libraries, Part 2: Searching for packages](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-2)
+- [Exploring the NuGet v3 Libraries, Part 3: Installing packages](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-3)
```

Comment:
Do we want these in the official docs? The disclaimer above it helpful, but these may be out of date already.

Reply:
Good point. I think we should at the very least mention which package versions these posts were relevant to. I'll bring this up with the PMs. In the meantime, do you know what package version these are associated with?

Reply:
Added a note about using 3.4.3. This was the last stable release before the blog posts.